radis/test/gpu/test_gpu.py

```diff
@@ -41,17 +51,17 @@ def test_eq_spectrum_emulated_gpu(emulate=True, plot=False, *args, **kwargs):
             "GaussianBroadeningWarning": "ignore",
         },
     )
-    sf._broadening_method = "fft"
+    sf.params.broadening_method = "fft"
```

Comment:
@dcmvdbekerom `_broadening_method` is deprecated => we were actually using 'voigt'  here